custom/_legacy/pact/templates/pact/patient/pactpatient_careplan.html

```diff
@@ -1,6 +1,9 @@
 {% extends "pact/patient/pactpatient_base.html" %}
 {% load case_tags %}
+{% load hq_shared_tags %}
+
 {% block patient-tab-container %}
+<script src="{% new_static 'jquery-timeago/jquery.timeago.js' %}"></script>
```

Comment:
actually i just looked a little closer and it looks like the bootstrap2 template actually initializes timeago for all elements with the timeago class: https://github.com/dimagi/commcare-hq/blob/pact-fix/corehq/apps/style/templates/style/bootstrap2/base.html#L360

i'm  thinking i'll just add the same to the boostrap3 base template unless there's any objections - i'm not sure how to reliably find out all the pages that need this.